package.json

```diff
@@ -4,11 +4,12 @@
   "version": "0.0.0",
   "main": "dist/index.html",
   "scripts": {
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
+    "gulp": "gulp",
+    "start": "npm run gulp",
+    "build": "npm run gulp -- build",
+    "test": "npm run gulp -- test",
+    "serve": "npm run gulp -- serve",
```

Comment:
Right, I misread http://stackoverflow.com/questions/22224831/no-command-gulp-found-after-installation/22274098#22274098 and http://engineering.hobsons.com/2015/06/26/build-tools-vs-npm-scripts-why-not-both/

Will do a fix for this.